src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,1426 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What are these suppressions are for (quick summary)? Thanks

Reply:
I added some interfaces (`ISupportKeyedService`, `ServiceCollectionKeyedServiceExtensions`, `ServiceProviderKeyedServiceExtensions`) and methods. It seems that there is some suppressions that are not needed anymore though.